borg/archiver.py

```diff
@@ -611,12 +629,11 @@ def do_break_lock(self, args):
 
     helptext = {}
     helptext['patterns'] = textwrap.dedent('''
-        Exclusion patterns support two separate styles, fnmatch and regular
+        Exclusion patterns support three separate styles, fnmatch and regular
         expressions. If followed by a colon (':') the first two characters of
```

Comment:
that sounds a bit strange, first saying "three" and then only mentioning "fnmatch and regex".